engine/loadcreate.go

```diff
@@ -83,7 +83,7 @@ func (lg *LoadGenerator) Run() {
 			s := fnodes[wsapiNode].State
 			s.APIQueue().Enqueue(c)
 			s.APIQueue().Enqueue(r)
-			time.Sleep(time.Duration(800/top) * time.Millisecond) // spread the load out over 800ms + overhead
+			//time.Sleep(time.Duration(800/top) * time.Millisecond) // spread the load out over 800ms + overhead
```

Comment:
At what load did calling sleep here plus creating the messages cause the load generation to take more than a second? Are you sure you didn't encounter full queues? Maybe instead of using this sleep to distribute the load the ticker should fire 20 times or 100 time per second and have lots of small bursts instead of 1 second bursts? Maybe we should sleep under some load threshold and not under higher ones?

Reply:
I don't know why the sleep limited me to 72 tps.  But removing the sleep allowed me to go to 100 tps.

I am happy with reverting this change.